docs/accelerometer.rst

```diff
@@ -3,7 +3,10 @@ Accelerometer
 
 .. py:module:: microbit.accelerometer
 
-This module gives you access to the on-board accelerometer.
+This module gives you access to the on-board accelerometer. As a convenience
```

Comment:
I agree. We say somewhere else that such things are objects representing aspects of the device's hardware. It'd be confusing if we were to differ here.